controls/calendar/commands.md

```diff
@@ -17,7 +17,7 @@ The following commands enable the navigation between same level views:
 * `NavigateToNextViewCommand`(`ICommand`)&mdash;Gets the command that is executed when the user navigates to the next view.
 * `NavigateToPreviousViewCommand`(`ICommand`)&mdash;Gets the command that is executed when the user navigates to the previous view.
 
-## Commands for Navigating between Views
+## Commands for Navigating Between Views
```

Comment:
[Never capitalize prepositions](https://docs.telerik.com/style-guide/titles-and-headings#capitalization).
```suggestion
## Commands for Navigating between Views
```